tfhe/src/shortint/parameters/mod.rs

```diff
@@ -10,12 +10,28 @@ pub use crate::core_crypto::commons::parameters::{
     CiphertextModulus as CoreCiphertextModulus, DecompositionBaseLog, DecompositionLevelCount,
     GlweDimension, LweDimension, PolynomialSize,
 };
+use crate::shortint::ciphertext::PBSOrder;
 use serde::{Deserialize, Serialize};
 
 pub mod parameters_wopbs;
 pub mod parameters_wopbs_message_carry;
 pub(crate) mod parameters_wopbs_prime_moduli;
 
+#[derive(Debug, PartialEq, Eq, Copy, Clone, Serialize, Deserialize)]
+pub enum EncryptionKeyChoice {
+    Big,
+    Small,
+}
```

Comment:
This could use a small docstring, mainly to quicly explain the tradeoff between the possible choices

Reply:
agreed !

Reply:
feedback wanted on that docstring